source/includes/api/recognition/_nominations.md

```diff
@@ -0,0 +1,58 @@
+## Nominations
+
+### Update a Nomination's status
+
+An endpoint to update a status of a nomination for a given campaign. By passing
+a status parameter. An approver for the nomination must also be provided.
+You will be returned the updated JSON for the given nomination.
+
+``` http
+PATCH /api/v2/campaigns/:campaign_id/nominations/:id HTTP/1.1
+Authorization: Token token=xxx
+Content-Type: application/json
+
+{
+  "transition" : "approve",
+  "approver" : 123
+}
+```
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id" : "1",
+  "state" : "approved_l1",
+}
+```
+
+#### HTTP Request
+
+`PATCH /api/v2/campaigns/:campaign_id/nominations/:id`
+
+#### Attributes
+
+Attribute | Type | Info
+--------- | ---- | ----
+id | integer | (required) nomination id
+approver | integer | (required) the approver's user id
+campaign_id | integer | (required) campaign id
+reason | string | (optional) string which can be provided when declining a nomination
+transition | string | (required) a valid transition to be performed on the award (approve or decline only)
+
+### Approver Information
+
+The Approver passed into the API needs to be a valid approver for the
+nomination and for the approval level.
```

Comment:
Reword: "The approver should be a valid approver for the nomination, and for the approval level."?